<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QDialog" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zaloguj się</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nazwa użytkownika:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hasło:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Zaloguj</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="logInLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Zaloguj się&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Zarejestruj się</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="alertBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
font-weight: bold;
font-size: 15px;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
